test/integration/test_generate_specs.py

```
@@ -79,12 +77,14 @@ def test_generate_specs_max_min() -> None:
                         get_min_max_src.add(fn.get_source_code_with_specs())
 
-
-            assert len(get_min_max_src) > 0, f"Expected at least one verified specification for 'get_min_max'"
+            assert len(get_min_max_src) > 0, (
+                "Expected at least one verified specification for 'get_min_max'"
+            )
             assert next(iter(get_min_max_src)) == VERIFIED_FUNCTION_SRC_CODE
     finally:
         # Clean up the temporary proof states.
         cmd = f"rm {PATH_TO_INTEGRATION_TEST_DIR}/*.pkl"
         result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)